cime_config/cam.case_setup.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python3
+
+"""Copy GEOS-Chem configuration files from source to the case directory.
+This script is run from CIME when calling case.setup"""
+
+import logging
+import os
+import shutil
+import sys
+
+_CIMEROOT = os.environ.get("CIMEROOT")
+if _CIMEROOT is None:
+    raise SystemExit("ERROR: must set CIMEROOT environment variable")
+# end if
+_LIBDIR = os.path.join(_CIMEROOT, "CIME", "Tools")
+sys.path.append(_LIBDIR)
+sys.path.insert(0, _CIMEROOT)
+
+#pylint: disable=wrong-import-position
+from CIME.case import Case
```

Comment:
Is this and the above to CIMEROOT all for getting case for the logger and cam options?

Reply:
Yes. Most of CIME works off of a case object and this is the standard way to bootstrap one of those.